config/locales/pt-BR.yml

```diff
@@ -0,0 +1,175 @@
+        status: "Status"
+        uptime: "Uptime"
+        source_code: "Código Fonte"
+        data: "Dump de Dados"
+        discussion_forum: "Fóruns"
+        statistics: "Estatísticas"
+        blog: "Blog"
+        about: "Sobre o Rubygems"
+        help: "Ajuda"
+        api: "API"
+        guides: "Guias"
+        contribute: "Como Contribuir"
+        supported_by: "Patrocínio"
+        hosted_by: "Hospedagem"
+        designed_by: "Design"
+        resolved_with: "DNS"
+        optimized_by: "Otimizações"
+        tracking_by: "Coleta de Dados"
+        monitored_by: "Monitoramento"
+        gems_served_by: "Provisionamento"
+
+  pages:
+    about:
+      title: Sobre o Rubygems
+      purpose:
+        header: "Bem vindo ao %{site}, a comunidade do serviço de hospedagem de gems. Este projeto tem três propósitos:"
+        better_api: Prover uma API melhor para lidar com gems
+        transparent_pages: Criar páginas mais acessíveis e transparentes para os projetos
+        enable_community: Empoderar a comunidade para melhorar o site
+      founding: "Este projeto foi iniciado em Abril de 2009 por %{founder}, e desde então tem crescido para incluir as contribuições de %{contributors} e %{downloads}. A partir do lançamento do RubyGems 1.3.6 o site foi renomeado de Gemcutter para %{title} para solidificar o papel centra do site na comunidade Ruby."
+      support: "Pelo Gemcutter não ser mantido por uma empresa específica, o projeto contou e conta com a contribuição de muitas pessoas. O design atual, ilustrações e o frontend do site foram criados pela %{dockyard}. Nós também contamos com ajuda do %{github} por nos ajudar compartilhar código facilmente. Este site foi inicialmente hospedado no %{heroku}, que nos ajudaram a validar o Gemcutter como uma solução viável para toda a comunidade Ruby."
+      technical: "Alguns aspectos técnicos sobre este site: 100% Ruby. A parte principal é construída como um aplicativo %{rails}, e as gem são servidas via %{sinatra}. As gems são hospedadas no %{s3}, e o tempo de espera entre publicar uma nova gem e disponibilidade para instalação é mínimo. Para mais informações, verifique o %{source_code}, com a licença %{license} no GitHub."
+
+  passwords:
+    edit:
+      title: Resetar senha
+      submit: Salvar senha
+    new:
+      title: Alterar senha
+      will_email_notice: Você vai receber um email com o link para atualizar sua senha.
+      submit: Resetar senha
+
+  profiles:
+    edit:
+      title: Editar Perfil
+      hide_email: Não mostrar meu email
+      reset_password:
+        title: Resetar senha
+        text: "Esqueceu sua senha? Sem problema. %{link}"
+        link_text: Solicitar uma nova senha.
+      api_access:
+        title: Acesso à API
+        key_is: "A chave da sua API é %{key}."
+        credentials: "Se você quiser usar as %{gem_commands_link} pela linha comando, você vai precisar de um arquivo %{gem_credentials_file}. Para gerar este arquivo rode o seguinte comando:"
```

Comment:
The result would like `Se você quiser usar as comandos pela linha comando`.

I would replace `Se você quiser usar as %{gem_commands_link} pela linha comando` with `Se você quiser usar %{gem_commands_link} pela linha de comando`, and then change `link_text` as well.